group-aws/README.md

```diff
@@ -0,0 +1,26 @@
+# Amazon Web Services
+
+The Amazon Web Services Working Group focuses on delivering modules
+and features to Ansible to support working with Amaazon Web Services (AWS)
+
+## Meetings
+
+There are currently no regular meetings held.  The IRC channel is the main and official place to countact the members. 
+
+## Collaboration
+
+Apart from IRC The AWS Working Group collaborates via tickets and pull
+requests in the main Ansible repository.
+* [Tickets](https://github.com/ansible/ansible/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20aws)
+* [Pull-requests](https://github.com/ansible/ansible/pulls?q=is%3Apr+is%3Aopen+aws)
+
+## Leads
+Leads are still to be agreed.
```

Comment:
It would be useful to have a lead that has merge access and can close tickets.
But we have not set any rules to this yet.